pages/common/xml-validate.md

```diff
@@ -0,0 +1,24 @@
+# xml-validate
+
+> Validate XML documents.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- Validate XML document for well-formedness only (default):
+
+`xml validate --well-formed {{path/to/input1.xml|URI}} {{input2.xml ...}}`
+
+- Validate XML document against Document Type Definition (DTD):
+
+`xml validate --dtd {{path/to/schema.dtd}} {{path/to/input1.xml|URI}} {{input2.xml ...}}`
+
+- Validate XML document against XML Schema Definition (XSD):
+
+`xml validate --xsd {{path/to/schema.xsd}} {{path/to/input1.xml|URI}} {{input2.xml ...}}`
+
+- Validate XML document against a Relax NG schema (RNG):
```

Comment:
Perhaps this will read better?
```suggestion
- Validate an XML document against a Relax NG schema (RNG):
```

Reply:
xml-validate.md:> Validate XML documents.
xml-validate.md:- Validate XML document(s) for well-formedness only:
xml-validate.md:- Validate XML document(s) against a Document Type Definition (DTD):
xml-validate.md:- Validate XML document(s) against an XML Schema Definition (XSD):
xml-validate.md:- Validate XML document(s) against a Relax NG schema (RNG):